### DISCUSSION TITLE
同一組輸入TTS模塊和SSML的行為不同?

### DISCUSSION THREAD

**Discussion by hymch** [lenML/Speech-AI-Forge#156] [讨论 | General]
使用其間觀察如下, 也不知是一個還是多個問題的混合, 所以到列寫一下

1. TTS-ChatTTS 的Style生成, 經常只輸出幾秒內接近空白的結果 
2. 使用Builder生成spk1.json文件, 以上情況更多發
3. 但同一段輸入, 於SSML使用spk1.json加上style生成則有多數能正常輸出, 但也有吞輸入的問題, 特別是[laugh] 基本完全丟失

於1. 2. 時, 如何下調temp有時能多成功幾次, 但更多時候是報"summary<=0"的error而不能運行

**Comment by zhzLuke96** [comment]
没有可复现的流程参数没法确定具体是什么问题

我大概猜测应该和代码无关，而是模型目前就是这个生成质量，吞字漏字现在都是没法完全杜绝的问题，特别是注入特定的 prompt 比如 style 和 参考音频 都会继续进一步降低生成质量，错误的输出结束 token、空白token 都是生成质量差的表现
